src/viam/examples/modules/tflite/main.cpp

```diff
@@ -750,34 +743,15 @@ int serve(const std::string& socket_path) try {
     vsdk::Registry::register_model(module_registration);
 
     // Construct the module service and tell it where to place the socket path.
-    auto module_service = std::make_shared<vsdk::ModuleService_>(socket_path);
-
-    // Construct a new Server object.
-    auto server = std::make_shared<vsdk::Server>();
+    auto module_service = std::make_shared<vsdk::ModuleService>(socket_path);
```

Comment:
I've left the tflite module as an example of using the "old" constructor (`ModuleService(std::string socket_path)`), and doing arg checking + model checking manually. Signal handling and manual calls to the underlying `Server` should no longer be necessary.